task/sast-snyk-check/0.3/README.md

```diff
@@ -0,0 +1,39 @@
+# sast-snyk-check task
+
+## Description:
+
+The sast-snyk-check task uses Snyk Code tool to perform Static Application Security Testing (SAST) for Snyk, a popular cloud-native application security platform.
+
+Snyk's SAST tool uses a combination of static analysis and machine learning techniques to scan an application's source code for potential security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks.
+
+> NOTE: This task is executed only if the user provides a Snyk token stored in a secret in their namespace. The name of the secret then needs to be supplied in the `snyk-secret` pipeline parameter.
+
+## Params:
+
+| name              | description                                                                                      | default value | required |
+|-------------------|--------------------------------------------------------------------------------------------------|---------------|----------|
+| SNYK_SECRET       | Name of secret which contains Snyk token.                                                        | snyk-secret   | yes      |
+| ARGS              | Append arguments.                                                                                | ""            | no       |
+| IMP_FINDINGS_ONLY | Report only important findings. Default is true. To report all findings, specify "false"         | true          | yes      |
+| KFP_GIT_URL       | Link to the known-false-positives repository. If left blank, results won't be filtered           | ""            | no       |
+| PROJECT_NVR       | Name-Version-Release (NVR) of the scanned project, used to find path exclusions (it is optional) | ""            | no       |
+| RECORD_EXCLUDED   | Write excluded records in file. Useful for debugging (it is optional). Default is "false"        | false         | no       |
```

Comment:
This doesn't match the description from the task parameter. I will update it in a bit...